docs/update_release_notes.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Script to help with the SMQTK release process. Performs the following steps:
+#   - Poetry version (major, minor, or patch)
+#   - Rename release_notes/pending_release file to release_notes/version
+#   - Add reference to new release notes file in release_notes.rst
+#   - Add new release notes stub file
+#
+# Two git commits are created. One for the version bump and one for the new
+# release notes stub file.
+#
+
+# Check args
+if [ $# != 1 ]
+then
+  echo "Please enter valid version bump type. Options: major, minor, or patch"
+  exit 1
+fi
+
+if [ $1 != 'major' ] && [ $1 != 'minor' ] && [ $1 != 'patch' ]
+then
+  echo "Please enter valid version bump type. Options: major, minor, or patch"
+  exit 1
+fi
+
+RELEASE_NOTES_PATH="release_notes"
+RELEASE_TYPE=$1
```

Comment:
Its good practice to bound things like this in quotes, partially for functionality and partially for input sanitization reasons
```suggestion
RELEASE_TYPE="$1"
```
Same goes for some other assignment instances below (e.g. L33).

Reply:
Also this specific variable doesn't look like it's being used anywhere?